keyboards/sofle/keymaps/flare576/config.h

```diff
@@ -0,0 +1,38 @@
+/* Copyright 2021 Flare576 <flare576@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+#define ENCODER_DIRECTION_FLIP
+#define OLED_TIMEOUT 10000
+
+#define MOUSEKEY_INTERVAL 16
+#define MOUSEKEY_DELAY 0
+#define MOUSEKEY_TIME_TO_MAX 60
+#define MOUSEKEY_MAX_SPEED 7
+#define MOUSEKEY_WHEEL_DELAY 0
+
+#ifdef ENCODER_RESOLUTION
+    #undef ENCODER_RESOLUTION
+    #define ENCODER_RESOLUTION 4
+#endif
+#define TAPPING_TERM_PER_KEY
+#ifdef TAPPING_TERM
+    #undef TAPPING_TERM
+    #define TAPPING_TERM 200
+#endif
+#ifdef MANUFACTURER
+    #undef MANUFACTURER
+    #define MANUFACTURER Flare576
+#endif
```

Comment:
```suggestion
#undef MANUFACTURER
#define MANUFACTURER Flare576
```

Reply:
Just checking my understanding (I'm new to C as well as to QMK) - we can do this without the `if` check because it absolutely MUST be defined by the time this code is being evaluated?

Reply:
partially.  It also has no issues if it's not defined.